src/components/inputs/number/ohri-number.component.tsx

```diff
@@ -102,6 +102,7 @@ const OHRINumber: React.FC<OHRIFormFieldProps> = ({ question, onChange, handler
             className={isFieldRequiredError ? styles.errorLabel : ''}
             warn={warnings.length > 0}
             warnText={warnings[0]?.message}
+            step="any"
```

Comment:
Shouldn't this correspond to a [number](https://react.carbondesignsystem.com/?path=/docs/components-numberinput--playground)?

Reply:
Nice catch @denniskigen, I've used a more specific step that allows up to 2 decimal points.